client/blocks/reader-author-link/index.jsx

```diff
@@ -29,7 +29,7 @@ const ReaderAuthorLink = ( { author, post, siteUrl, children } ) => {
 	}
 
 	return (
-		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" onClick={ recordAuthorClick }>
+		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" rel="noopener noreferrer" onClick={ recordAuthorClick }>
```

Comment:
I had the same thought, but I'm not sure our new lint rule (https://github.com/Automattic/eslint-config-wpcalypso/pull/2) is smart enough to figure that out :) Might mean peppering the code with eslint exception comments?